casper-finality-proofs/src/compute_shuffled_index/compute_shuffled_index_helpers.rs

```diff
@@ -0,0 +1,100 @@
+use plonky2x::{prelude::{U64Variable, Bytes32Variable, PlonkParameters, CircuitBuilder, ByteVariable, BytesVariable, CircuitVariable, Variable, BoolVariable}, frontend::vars::EvmVariable};
+use plonky2::field::types::Field;
+use itertools::Itertools;
+use crate::utils::{utils::exp_from_bits, variable::bits_to_variable};
+
+/// Returns the first 8 bytes of the bytes concatenation of seed and current_round as U64Variable
+pub fn compute_hash<L: PlonkParameters<D>, const D: usize>(
+    builder: &mut CircuitBuilder<L, D>,
+    seed: Bytes32Variable,
+    current_round: usize
+) -> U64Variable {
+    let const_0_byte: ByteVariable = ByteVariable::constant(builder, 0);
+    let current_round_bytes: ByteVariable = ByteVariable::constant(builder, current_round as u8);
+
+    let mut hash: BytesVariable<33> = BytesVariable([const_0_byte; 33]);
```

Comment:
The name is misleading, this is not a hash, this is the thing to be hashed. Rename this to something like "concatenation"